README.md

```diff
@@ -26,52 +26,61 @@ This is a NodeJS project with Express, GraphQL, PSQL and Auth0 login
 
 ### Database
 
-Currently we use MongoDB (until it is changed)
+This repo is currently using a local PostgreSQL instance
 
-The repo won't run locally if you don't set up a basic mongodb.
-You can do it in your local checkout; the /data directory is ignored by gitignore.
+This is temporarily until docker is set up
 
-If you've never done this before
-brew services stop mongodb
-brew uninstall mongodb
+You will need to set up a local [PostgreSQL db](https://www.postgresql.org/download/) to connect to the db
 
-brew tap mongodb/brew
-brew install mongodb-community
-brew services start mongodb-community
-sudo mkdir -p data/db
-
-Then
-mongod --dbpath data/db
-in a separate window
-mongo
-
-then in the mongo interface
-use etherpay
+The application is currently using [bookshelf.js](https://bookshelfjs.org/) and [knex.js](http://knexjs.org/)
 
 ---
 
-### Installation
+### GraphQL
 
-`npm install`
+Testing GraphQL querys/mutations can be done through [GraphIQL](https://graphql.org/learn/queries/)
```

Comment:
shouldn't this be a link to our `localhost:xxxx/graphiql`?

Reply:
I was pointing it at the documentation so poeple could see how to build the query. I'll update it so I have both and its a little more clear